app-compat-travis.yml

```diff
@@ -0,0 +1,22 @@
+# Test compatibility with applications written in ksh
+sudo: required
+
+services:
+    - docker
+
+matrix:
+    include:
+        - env:
+            - $PROJECT_NAME=kshdb
+              GIT_URL="https://github.com/rocky/kshdb.git"
+              COMMANDS="ksh ./autogen.sh; make test"
+
+script:
+    - docker pull fedora
+    - docker run fedora bash -c "
+      dnf install -y 'dnf-command(copr)';
+      dnf copr enable -y @ksh/latest;
```

Comment:
Just curious, but does this use the latest OBS build or something else? Perhaps a comment somewhere in this file to explain which version of ksh is being used would be a good idea.

Reply:
It uses builds from [copr](https://copr.fedorainfracloud.org/coprs/g/ksh/latest/). I added a comment mentioning it.